src/shared/components/case-editor/case-event-completion/components/case-event-completion-task-reassigned/case-event-completion-task-reassigned.component.ts

```diff
@@ -32,7 +32,7 @@ export class CaseEventCompletionTaskReassignedComponent implements OnInit, OnDes
   public ngOnInit(): void {
     // Get case id and task from the parent component
     this.caseId = this.parentComponent.context.caseId;
-    const task = this.parentComponent.context.task;
+    const task = this.parentComponent.context.reassignedTask;
```

Comment:
very minor but you could use destructuring here to neaten things up:

```
const { caseId, reassignedTask } = this.parentComponent.context;
this.caseId = caseId;
const task = reassignedTask
```

Reply:
Done